ui/src/containers/AlertPage.js

```diff
@@ -281,15 +274,19 @@ export default function AlertPage() {
   );
 
   return (
-    <AlertPageContainer>
-      <AlertPageHeader
-        activeAlerts={leafAlerts.length}
-        critical={criticalAlerts.length}
-        warning={wariningAlerts.length}
-      />
-      <AlertContent>
-        <ActiveAlertTab data={leafAlerts} columns={columns} />
-      </AlertContent>
-    </AlertPageContainer>
+    <AppContainer hasPadding>
+      <AppContainer.OverallSummary>
+        <AlertPageHeader
+          activeAlerts={leafAlerts.length}
+          critical={criticalAlerts.length}
+          warning={wariningAlerts.length}
+        />
+      </AppContainer.OverallSummary>
+      <AppContainer.MainContent hasPadding>
+        <AlertContent>
+          <ActiveAlertTab data={leafAlerts} columns={columns} />
```

Comment:
I checked the reason we have the wrong non-selected color for the table row is because we didn't set correctly the `backgroundVariant` in the table.

```js
        <Table.SingleSelectableContent
          rowHeight="h48"
          separationLineVariant="backgroundLevel1"
          backgroundVariant="backgroundLevel3" // was wrongly set to backgroundLevel2 before
          customItemKey={(index, data) => {
            return data[index].id;
          }}
        />
```